src/pages/train/StepViewer.ts

```diff
@@ -70,6 +72,10 @@ export class StepViewer {
     this.#element.setAttribute('code', value);
   }
 
+  set allowBackgroundExecution(value: string | null) {
+    this.#element.setAttribute('allow-background-execution', value!);
```

Comment:
This should be `this.#element.toggleAttribute('allow-background-execution', bool true | false)`

`CodeEditor` doesn't check the value of `allow-background-execution`, it merely checks if it **has** the attribute (`hasAttribute("allow-background-execution")`)